[blender] Simplify and clarify support policy (#6958)
captn3m0/endoflife.date: products/blender.md

-   releaseCycle: "3.6"
    lts: true
    releaseDate: 2023-06-27
    eoas: 2023-11-14
    eol: 2025-06-27
    latest: "3.6.21"
    latestReleaseDate: 2025-02-17

-   releaseCycle: "3.5"
    releaseDate: 2023-03-29
    eoas: 2023-06-27
    eol: 2023-06-27
    latest: "3.5.1"
    latestReleaseDate: 2023-04-25

-   releaseCycle: "3.4"
    releaseDate: 2022-12-07
    eoas: 2023-03-29
    eol: 2023-03-29
    latest: "3.4.1"
    latestReleaseDate: 2022-12-20

-   releaseCycle: "3.3"
    lts: true
    releaseDate: 2022-09-07
    eoas: 2022-12-07

>     eol: 2024-09-01

Author: ```suggestion
    eol: 2024-09-07
```

-   releaseCycle: "2.93"
    lts: true
    releaseDate: 2021-06-02
    eoas: 2021-12-03

>     eol: 2023-06-01

Author: ```suggestion
    eol: 2023-06-02
```

> entirety of the 3D pipeline—modeling, rigging, animation, simulation, rendering, compositing and
> motion tracking, video editing and 2D animation pipeline.

[Since February 2023](https://code.blender.org/2023/02/blender-release-cycle-update/#new-release-schedule),

> Blender follows a 3-releases per year schedule, with one of the releases offering long-term support (Blender LTS).

Reviewer: ```suggestion
Blender follows a 3-releases-per-year schedule, with one of the releases offering long-term support (Blender LTS).
```

> Non-LTS releases are supported until the next release with high priority bugs fixes. High severity and normal bugs

Reviewer: ```suggestion
Non-LTS releases are supported until the next release with high-priority bug fixes. High severity and normal bugs
```

may be fixed too if agreed on by the team.

LTS releases are supported for 2 years. At first the support level is the same as non-LTS releases. But after the next

> release is out they are supported with critical bug-fixes only.

Reviewer: Since these lines are being rewritten, we might as well apply [semantic line-breaking](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) to make reading the source (and future edits) easier:

```
Non-LTS releases are supported until the next release with high priority bugs fixes.
High severity and normal bugs may be fixed too if agreed on by the team.

LTS releases are supported for 2 years. At first the support level is the same as non-LTS releases.
But after the next release is out they are supported with critical bug-fixes only.
```

Author: So far the recommendation is to break at 100 characters, see https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide#text.

Reviewer: Fair enough. I might propose a change in a separate thread.

Author: Can you open a dedicated issue first, so the team can discuss about it ?

Reviewer: Yeah, that's what I meant, sorry for the ambiguity :)

Reviewer: Opened https://github.com/endoflife-date/endoflife.date/discussions/6975.

> LTS releases are supported for 2 years. At first the support level is the same as non-LTS releases. But after the next
> release is out they are supported with critical bug-fixes only.

Reviewer: ```suggestion
LTS releases are supported for 2 years. At first the support level is the same as non-LTS releases; but after the next
release is out, they are supported with critical bugfixes only.
```
